cmd/system/node.go

```diff
@@ -0,0 +1,138 @@
+package system
+
+import (
+	"fmt"
+	"github.com/alexellis/arkade/pkg/archive"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/get"
+	cp "github.com/otiai10/copy"
+	"github.com/spf13/cobra"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func getLatestVersion(version, channel string) (*string, error) {
+	resp, err := http.Get(fmt.Sprintf("https://nodejs.org/download/%s/%s", channel, version))
```

Comment:
As a nit, generally we have used `res` elsewhere, can you be consistent?

Reply:
done, renamed to `res`

Reply:
I'm still seing it as `resp`?

Reply:
Done